Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -51,8 +51,8 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
+        'Geom' : 'Extended2017dev',
+        'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
> This shouldn't "interfere" with the new pixel geometry PRs, as those are creating a separate workflow for validation.

This is not strictly correct, the new WFs exist to evaluate the new *FPIX* individually. There are aso *BPIX* changes that need validation in the normal 2017 geometry. However, these are not as drastic as the FPIX changes and might be fine to coexist with the HCAL changes.